limited/cursebearer

```diff
@@ -1,18 +1,20 @@
 **Cursebearer** | Townsfolk Power | Limited
 __Basics__
-The Cursebearer is a member of the townsfolk team who imitates and is seen as a wolf role.
+The Cursebearer is a member of the townsfolk team who is linked to an arbitrary werewolf.
 __Details__
-The Cursebearer imitates another in-play werewolf team role. They are strongly disguised as that role and are a lycan if that role is. Additionally, they can use all the powers that role can use, with some exceptions: any power that would result in joining the wolfpack or in a role change fails; any power that would result in the Cursebearer's role changing instead results in a loss of power as described below.
-When the last player that originally started out with the role that the Cursebearer is imitating dies, the Cursebearer is informed and loses their power.
-When the Cursebearer loses their power they will not be able to use any of their abilities anymore, but they will keep their disguise and they will still be seen as a lycan if they were imitating a lycan.
+The Cursebearer is linked to an arbitrary werewolf-aligned member of the wolfpack. They are strongly disguised as that player's role, and they are a lycan if that role is. The Cursebearer knows what role they are disguised as, but they do not know who the player they are linked to is.
+When the werewolf they are linked to dies, the Cursebearer is informed. 
 
 __Simplified__
-The Cursebearer is a member of the townsfolk team who imitates and is seen as a in-play werewolf team role. They may use the powers of this role.
-If no players with that role remain, the Cursebearer can't use its powers anymore, but retains the disguise.
+__Basics__
+The Cursebearer is a member of the townsfolk team who is disguised as an arbitrary werewolf. If the werewolf they are disguised as dies, the Cursebearer is informed.
 
 __Formalized__
 Role Attribute: `Lycan`
-Starting: Strongly Disguise @Self as %Role%
+Starting: 
+  • Target %Role% (Role)
+  • Apply `CursebearerLink` to @Target->RandomPlayer
```

Comment:
There is absolutely no way this works, right? Is there?

Reply:
According to [WWRF](https://github.com/WerewolvesRevamped/Werewolves-Roles/blob/main/WWRF%20Guide.md#role-type) roles support a "Players" property access, but not a "RandomPlayer" one